src/features/hmDashboard/HmDashboardScreen.tsx

```diff
@@ -0,0 +1,42 @@
+import React, { memo, useEffect } from 'react'
+import { BottomSheetModalProvider } from '@gorhom/bottom-sheet'
+
+import { useAnalytics } from '@segment/analytics-react-native'
+import { WebView } from 'react-native-webview'
+
+import Box from '../../components/Box'
+
+import { AppScreens } from '../../utils/analytics/screens'
+
+const HmDashboardScreen = () => {
+  const { screen } = useAnalytics()
+
+  // Segment log screen visit
+  useEffect(() => {
+    screen(AppScreens.HMDASHBOARD)
+  }, [screen])
```

Comment:
We track here.

Reply:
Isn't this tracking tab load. Not link clicking?

Reply:
This is tracking screen visit in the app. The link clicks are being tracked at hm-dashboard

Reply:
How exactly is hm-dashboard tracking these inbound clicks? UTM? Segment?

Reply:
When the user go to the HM dashboard screen, it get's logged to segment. I am not sure the UTM situation is applicable here. The users are still within the app although accessing the dashboard.  I have added the UTM tracking though

Reply:
We don't just want to track number of visits to the dashboard, but we need to be able to differentiate which of those are coming from the app. More specifically, we want to know how many registrations and license purchases come following a redirection from the app. Can you figure out the best way for us to measure that? Let me know if you need additional access.

Reply:
1. Don't you think a different issue needs to be drafted for this?
2. The issue i see is the app track users based on their wallet id whiles the dashboard uses their user id. So we need to agree on a common identifier to be able to create a linkage between the 2.